src/fmu/dataio/datastructure/configuration/global_configuration.py

```diff
@@ -152,7 +168,5 @@ def roundtrip(obj: Dict) -> Dict:
     """
     return GlobalConfiguration.model_validate(obj).model_dump(
         mode="json",
-        exclude_defaults=True,
         exclude_none=True,
-        exclude_unset=True,
```

Comment:
Dropping defaults and unset fields is not desired behavior, and lead to unwanted effects further in the execution. This is removed.